src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -327,7 +324,7 @@ ApiGlobe.prototype.createSceneGlobe = function createSceneGlobe(coordCarto, view
 
     var coordinate = new C.EPSG_4326(coordCarto.longitude, coordCarto.latitude, coordCarto.altitude);
 
-    this.scene = Scene(coordinate, viewerDiv, debugMode, gLDebug);
+    this.scene = Scene('EPSG:4978', coordinate, viewerDiv, debugMode, gLDebug);
```

Comment:
In this case, I need to think about it. Maybe then `EPSG:4978` is simply and plainly incorrect and I need to wait for #246 to get merged for this to make any sense. I won't but a `// warning: this is wrong` in the code :-) 

At any rate, no time for this now, I'll see next week.